cli/src/deploy/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod deploy;
```

Comment:
do we need to have this second-inner `deploy` module? we can also re-export things ie `pub use deploy::new`